site/src/modules/workspaces/WorkspaceTiming/WorkspaceTimings.tsx

```diff
@@ -135,19 +158,18 @@ export const WorkspaceTimings: FC<WorkspaceTimingsProps> = ({
 							)}
 
 						{view.name === "detailed" &&
-							view.category.id === "workspaceBoot" && (
+							view.stage.section !== "provisioning" && (
 								<ScriptsChart
 									timings={agentScriptTimings
-										.filter((t) => t.stage === view.stage)
+										.filter((t) => t.stage === view.stage.name)
 										.map((t) => {
 											return {
-												range: extractRange(t),
+												range: toTimeRange(t),
 												name: t.display_name,
 												status: t.status,
 												exitCode: t.exit_code,
 											};
 										})}
```

Comment:
could be an implicit return

```suggestion
										.map((t) => ({
											range: toTimeRange(t),
											name: t.display_name,
											status: t.status,
											exitCode: t.exit_code,
										}))
```